codebase/superdarn/src.bin/tk/tool/trim_fit.1.20/trim_fit.c

```diff
@@ -292,36 +298,36 @@ int main (int argc,char *argv[]) {
       if ((cpid !=-1) && (prm->cp !=cpid)) continue;
       if ((set_channel==0) &&
           (channel !=-1) && (prm->channel !=channel)) continue;
- 
+
       if (set_channel==1) prm->channel=channel;
 
       if ((inxfp !=NULL) && (irec==1)) OldFitInxHeaderFwrite(inxfp,prm);
       dnum=OldFitFwrite(fp,prm,fit,NULL);
       if (inxfp !=NULL) OldFitInxFwrite(inxfp,drec,dnum,prm);
       atime=TimeYMDHMSToEpoch(prm->time.yr,
-	  	      prm->time.mo,
+                      prm->time.mo,
```

Comment:
Any reason you change a bunch of spacing? maybe it should be aligned with `prm->time.yr,`?